packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -65,7 +74,11 @@ export default class QuanticRefineModalContent extends LightningElement {
   @api disableDynamicNavigation = false;
 
   /** @type {object} */
-  data;
+  facetData;
+  /** @type {boolean} */
+  isSortComponentReady = false;
+  /** @type {object} */
+  sortData;
```

Comment:
Really small detail but it helps a bit if these two params are grouped together :) 
```suggestion
  facetData;
  /** @type {object} */
  sortData;
  /** @type {boolean} */
  isSortComponentReady = false;
```

Reply:
You don't have to change it if you prefer not to, it's just nitpicking.